README.md

```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
     }
   })
   console.log(dimensions);
-
   await browser.close();
 })();
 ```
 
-## Contributing
+#### Intercept requests and wait for navigation
+Playwright can intercept network requests from a page to [modify or stub](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-request) them. Playwright scripts can also wait for [page navigations](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitfornavigationoptions), [selectors](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options), [network responses](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforresponseurlorpredicate-options), and [arbitrary predicates](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforfunctionpagefunction-options-args). This snippet intercepts network requests from a Firefox page, interacts with the page and waits for a navigation.

```

Comment:
nit: This should link to the `page.route` once we push 0.11

Reply:
opting to keep the readme relevant for 0.10 for now, and bring 0.11 changes together when we release